src/main/kotlin/app/revanced/patches/nyx/misc/pro/fingerprints/UnlockProFingerprint.kt

```diff
@@ -0,0 +1,15 @@
+package app.revanced.patches.nyx.misc.pro.fingerprints
+
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import app.revanced.patches.nyx.misc.pro.annotations.UnlockProCompatibility
+
+@Name("unlock-pro")
```

Comment:
Apply this name to the fingerprint as well. The names should somewhat convey what the fingerprint resolves to.
```suggestion
@Name("check-pro-fingerprint")
```